app/models/local_transaction_service.rb

```diff
@@ -0,0 +1,18 @@
+require "singleton"
+
+class LocalTransactionService
```

Comment:
```suggestion
class LocalTransactionServices
```

As it can represent multiples service

Reply:
Renamed as requested